framework/src/media/MediaPlayerImpl.cpp

```diff
@@ -405,18 +405,23 @@ player_result_t MediaPlayerImpl::setVolume(int vol)
 	return ret;
 }
 
-void MediaPlayerImpl::setVolumePlayer(int vol, player_result_t &ret)
+void MediaPlayerImpl::setVolumePlayer(uint8_t vol, player_result_t &ret)
 {
-	int vol_max = -1;
-	medvdbg("MediaPlayer Worker : setVolume\n");
+	uint8_t vol_max;
+	medvdbg("MediaPlayer Worker : setVolume %d\n", vol);
+
+	audio_manager_result_t result = get_max_audio_volume(&vol_max);
```

Comment:
Same as Above. (2 options.)

Reply:
Now the default value of the audio device is returned though the exclude config is set in getVolumePlayer().